12/umbraco-vendr/core/how-to-guides/migrating-from-vendr-to-umbraco-commerce.md

```diff
@@ -0,0 +1,319 @@
+---
+description: Learn how to migrate a Vendr solution to Umbraco Commerce.
+---
+
+# Migrate a Vendr solution to Umbraco Commerce
+
+This guides provides a step-by-step approach to migrating a default Vendr solution to Umbraco Commerce.
+
+## Key changes
+
+Before outlining the exact steps, there are a few key changes to be aware of.
+
+These changes will dictate the steps to take in the process of migrating to Umbraco Commerce.
+
+### Project, Package, and Namespace changes
+
+| Vendr | Umbraco Commerce |
+| ----- | ---------------- |
+| Vendr.Common | Umbraco.Commerce.Common |
+| Vendr.Core | Umbraco.Commerce.Core |
+| Vendr.Infrastructure | Umbraco.Commerce.Infrastructure |
+| Vendr.Persistence | Umbraco.Commerce.Persistence |
+| Vendr.Persistence.Sqlite | Umbraco.Commerce.Persistence.Sqlite |
+| Vendr.Persistence.SqlServer | Umbraco.Commerce.Persistence.SqlServer |
+| Vendr.Umbraco | Umbraco.Commerce.Cms |
+| Vendr.Umbraco.Web | Umbraco.Commerce.Cms.Web |
+| Vendr.Web | Umbraco.Commerce.Web |
+| Vendr.Web.UI | Umbraco.Commerce.Web.StaticAssets |
+| Vendr.Umbraco.Startup | Umbraco.Commerce.Startup |
+| Vendr | Umbraco.Commerce |
+
+### C# Class changes
+
+* Namespace changes as documented above.
+* All classes containing the `Vendr` keyword are now updated to `UmbracoCommerce`.
+  * Examples: `IVendrApi` is now `IUmbracoCommerceApi` and `AddVendr()` is now `AddUmbracoCommerce()`.
+
+### JavaScript changes
+
+* All `vendr` modules have changed to `umbraco.commerce` modules.
+* All `vendr` prefixed directives, services and resources are new prefixed with `uc`.
+* All `vendr` prefixed events now follow this format: `Umbraco.Commerce.{target}.{action}`.
+
+### UI Changes
+
+* All static UI assets are served via a Razor Compiled Library (RCL) and are no longer found in the `App_Plugins` folder.
+* The folder within `App_Plugins` still exists for the static assets that are user configurable.
+* The folder with `App_Plugins` has been renamed from `Vendr` to `UmbracoCommerce`.
+* UI Config files have changed from `.js` files to `.json`.
+
+## Migration steps
+
+### Step 1: Replace dependencies
+
+In this first step, we will be replacing all existing Vendr dependencies with Umbraco Commerce dependencies.
+
+1. Remove any installed Vendr packages (including Payment Providers).
+
+```bash
+dotnet remove package Vendr
+```
+
+2. Take a backup of any Vendr specific templates and config files you will want to reuse.
+3. Delete the Vendr `App_Plugins` folder.
+
+```bash
+rmdir App_Plugins\Vendr
+```
+
+4. Install `Umbraco.Commerce`.
+
+```bash
+dotnet add package Umbraco.Commerce
+```
+
+5. Install Umbraco Commerce packages including payment providers.
```

Comment:
I changed the wording here a bit, but I'm still a little confused.
Are you supposed to install ALL packages and payment providers in this step?
Further down the article, there are more steps for installing the packages again - so I'm wondering if this should only be the core product? 🤔

Reply:
Actually, I've made a mistake in my last comments. This probably should stay the same and just say to remove the payment providers too, the section about migration payment providers is actually for people with custom payment provider implementations and how to upgrade those so that is different 🤦‍♂️